code/datums/brain_damage/brain_trauma.dm

```diff
@@ -15,6 +15,7 @@
 	var/random_gain = TRUE //can this be gained through random traumas?
 	var/resilience = TRAUMA_RESILIENCE_BASIC //how hard is this to cure?
 	var/clonable = TRUE // will this transfer if the brain is cloned?
+	var/modifies_speech = FALSE
```

Comment:
If you don't mind exchanging performance for ease of use you could make the base handle_speed() unregister the signal instead of this var